### ISSUE TITLE
st_Make_grid trouble ?

### ISSUE THREAD

**Issue by j3r3m1** [orbisgis/h2gis#1038]
I have met trouble with the st_make_grid function:
Try this one:
```sql
CREATE TABLE grid AS SELECT * FROM
    ST_MakeGrid('POLYGON((0 0, 2 0, 2 2, 0 0))'::GEOMETRY, 1, 1);
```
The column "the_geom" has no type (just java object). It is not cast as geometry anymore

**Comment by ebocher** [comment]
This unit test works 

```
@Test
    public void testST_MakeGridFromGeometry21() throws Exception {
        ResultSet rs = st.executeQuery("SELECT * FROM st_makegrid('POLYGON((0 0, 2 0, 2 2, 0 0 ))'::GEOMETRY, 1, 1);");
        rs.next();
        System.out.println(((Geometry)rs.getObject(1)));                
    }
```
Can you point me more information about the use of ST_MakeGrid function

**Comment by j3r3m1** [comment]
In the H2 DataBase, the type of the column "the_geom" is "JAVA_OBJECT". Impossible to manipulate neiher cast it as a geometry then

**Comment by ebocher** [comment]
This is due to a change in H2. I will look on it.

**Comment by SPalominos** [comment]
Should have been fixed with PR #1043, it will be available on our nexus soon

**Comment by j3r3m1** [comment]
ok solved
